komodo/fetch.py

```diff
@@ -60,13 +56,12 @@ def fetch(pkgs, repo, outdir=".", pip="pip"):
 
     if missingpkg:
         eprint("Packages requested, but not found in the repository:")
-        eprint("missingpkg: {}".format(",".join(missingpkg)))
+        eprint("missingpkg: " + ",".join(missingpkg))
```

Comment:
`eprint("missingpkg: {','.join(missingpkg)}")` ?

Reply:
That is a possibility, is there a reason to prefer one over the other (I don't really know in this case)?

Reply:
I thought that `f"{xxx}"` is what we are after :)

Reply:
I know that `.format()` should be avoided, but do not know a reason to avoid the `+` operator on strings in general.. (but happy to change, just don't know why..)

Reply:
https://stackoverflow.com/questions/59180574/string-concatenation-with-vs-f-string

Reply:
Ok, just keep it as is I would say. It's more like to pursue your `f-formatting` career to the pure perfection :)